src/server.rs

```diff
@@ -96,10 +96,14 @@ impl Server {
want to achieve a fast, secure and distributed key-value store accessible through an HTTP API, we also want to propose persistence, encryption, WebSocket streaming, replication and a lot of features. 🦀" /> <meta property="og:image" content="https://image" /> <meta property="og:url" content="https://lucid-kv.store" /> <meta property="og:site_name" content="Lucid KV" /> <meta property="og:type" content="article" /> <meta name="theme-color" content="\#1d1b26"> <link rel="icon" type="image/png" href="https://avatars3.githubusercontent.com/u/56896360" /> <title>Lucid KV | High performance and distributed KV store w/ REST API. 🦀</title> <style> body { background-color: #FFFFFF; font-family: "Inter UI", -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, "Helvetica Neue", Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol", "Noto Color Emoji"; margin: 0; font-size: 1rem; font-weight: 400; line-height: 1.5; color: #1d1b26; text-align: left; } .showcase { font-family: 'Noto Sans', sans-serif; display: flex; justify-content: center; align-items: center; flex-direction: column; height: 95vh; } .showcase img { height: 30vh; -webkit-filter: drop-shadow(5px 5px 5px rgba(17, 24, 20, .21)); filter: drop-shadow(5px 5px 5px rgba(17, 24, 20, .21)); animation: float 6s ease-in-out infinite; } .showcase h1 { font-weight: 900; font-size: 60pt; margin: 20px 20px 0; } .showcase h2 { font-size: 30pt; margin: 0; font-weight: 400; } .showcase p { font-size: 17pt; text-align: center; } .showcase div { margin-top: 30px; } .btn { border: 1px solid #1d1b26; padding: 12px 15px; color: #1d1b26; text-decoration: none; border-radius: 3px; margin-top: 15px; text-align: center; } .btn.fill { background-color: #1d1b26; color: #FFFFFF; } .showcase .btn:not(.fill) { margin-left: 10px; } @keyframes float { 0% { -webkit-transform: translatey(0px); transform: translatey(0px); } 50% { -webkit-transform: translatey(-20px); transform: translatey(-20px); } 100% { -webkit-transform: translatey(0px); transform: translatey(0px); } } @media (max-width: 991px) { .showcase { height: 100vh; } .showcase img { height: initial; width: 60vw; } .showcase h1 { font-size: 40pt; } .showcase p { font-size: 12pt; padding: 0 20px; } .showcase div { display: flex; margin-top: initial; flex-direction: column; } .showcase .btn:not(.fill) { margin-left: initial; } } </style> </head> <body> <div class="showcase"> <a href="https://github.com/lucid-kv/lucid" target="_blank"> <img src="https://avatars3.githubusercontent.com/u/56896360" alt="lucid-kv logo" /> </a> <h1>Lucid KV</h1> <p>Lucid is an high performance and distributed KV store<br />accessible through an HTTP API. 🦀</p> <div> <a class="btn fill" href="https://medium.com/@clintnetwork/lucid-an-http-key-value-store-c0e734586e26"><i class="fas fa-info-circle"></i>&nbsp; Read About Lucid KV</a> <a class="btn " href="https://docs.lucid-kv.store/"><i class="fas fa-book"></i>&nbsp; Official Documentation</a> </div> </div> </body></html>"#;
```

Comment:
Please put the welcome page into it's own file using `include_str`, there's no reason for a html file of this size to be in a single line here.

Reply:
Do you think we put it into `assets` folder?

Reply:
Yeah, it's probably fine to put it there since the WebUI is already there anyway.